apps/api/src/common/config/schemas.ts

```diff
@@ -18,13 +18,19 @@ const urlArraySchema = z
         message: "Must be a comma-separated list of valid URLs",
     });
 
+const urlArrayMapSchema = z.record(
+    z.union([z.coerce.number().int(), z.string().regex(/^\d+$/)]), // key: number or string number
```

Comment:
Super edge case but would the key `"04"` be validated with this ruleset? Should we invalidate a chain id starting with a `0`?

Reply:
i think it's fine, chain ids are numbers actually but JSON doesn't allow numbers as key type so if you write, "04" to us it's the same as "4"